cpu/em_cpupower.py

```diff
@@ -19,10 +19,11 @@
 from avocado import Test
 from avocado import main
 from avocado import skipIf
-from avocado.utils import process, distro, cpu
+from avocado.utils import process, distro
 from avocado.utils.software_manager import SoftwareManager
 
-IS_POWER_NV = 'PowerNV' in cpu._get_cpu_info()
+# TODO : Logic need to change when we have lib fix
+IS_POWER_NV = 'PowerNV' in open('/proc/cpuinfo', 'r').read()
```

Comment:
@PraveenPenguin can't we wait for lib fix ?

Reply:
@narasimhan-v not sure how much that will take as my ealier PR also i need to rework , i think we can have this in and later in on time we can change all applicable test in one commit